homeassistant/components/zha/sensor.py

```diff
@@ -540,6 +601,26 @@ def extra_state_attributes(self) -> dict[str, Any]:
                 attrs["status"] = str(status)[len(status.__class__.__name__) + 1 :]
         return attrs
 
+    @property
+    def native_value(self) -> StateType:
+        """Return the state of the entity."""
+        state = super().native_value
+        description = self._ENTITY_DESCRIPTION_MAP.get(
```

Comment:
Why do we do this lookup again? Isnt it enough in init? Use the existing entity description on the class.